src/azul/plugins/metadata/hca/transform.py

```diff
@@ -1022,8 +1021,14 @@ def _matrices(self, file: api.File) -> MutableJSON:
                 'version': file.manifest_entry.version,
                 'name': file.manifest_entry.name,
                 'size': file.manifest_entry.size,
+                # FIXME: Remove deprecated field 'matrix_cell_count'
+                #        https://github.com/DataBiosphere/azul/issues/3180
                 'matrix_cell_count': matrix_cell_count,
-                'source': Submitter.title_for_file(file),
+                'matrixCellCount': matrix_cell_count,
+                # FIXME: Remove deprecated field 'source'
+                #        https://github.com/DataBiosphere/azul/issues/3180
+                'source': file_source,
```

Comment:
You're not following my review comment here. In case you missed it, fine. If not, please don't. Instead explain the deviation in a reply.

However, I think in this case my new = old approach would be awkward so this can stay as is.

Reply:
I did not miss your comment, however it seems I misinterpreted it, at least when it came to dictionaries. When you said:

> and don't duplicate the initialization (however small) but instead create the backwards compatible entry like so
> self.source = self.fileSource

I understood the desire to avoid doing the same assignment twice (once for the new variable and once for the deprecated), and instead the deprecated variable should be assigned a copy of the new variable's value.

In the case of new & deprecated keys in a dictionary literal however this approach didn't seem possible without breaking it up and adding an unnecessary key lookup like:

```
foo = {
    'new_key': _file.get("file_source")
}
foo['old_key'] = foo['new_key']
```

So I thought the compromise of factoring out the duplicated value to a variable

`file_source = _file.get("file_source")`

and assigning that to the new and deprecated keys would accomplish the desired optimization.

Reply:
Hmmm, ok. The reasoning behind requesting

```
x.new = expression
# FIXME
x.old = x.new
```

instead of 

```
x.new = expression
# FIXME
x.old = expression
```

was to 1) avoid duplicating code that would then always have to be manually kept consistent, 2) to avoid a potentially expensive operation (`expression`) from being evaluated twice and 3) reduce the size of the diff of the commit that addresses the FIXME.

For dictionaries, the pattern looks like

```
x = {
    "new": expression
}
# FIXME
x['old'] = x['new']
```

instead of 

```
x = {
    'new': expression,
    # FIXME
    'old': expression
}
```

I'm ok with temporarily adding an O(1) dict lookup.

If there is no variable `x` because the literal is returned right away or passed to another function, the best approach seems to me to extract either `expression` or the literal into a variable. Extracting the literal into a variable achieves aims 1 and 2 at the expense of aim 3. Use you discretion in this case, and chose whichever would look better once the FIXME is resolved, considering that the resolution of the FIMXE is not likely to inline the extracted variable. If both extractions (`expression` or literal) look equally ugly, stick with your original approach of duplicating the literal entry.